.kokoro/release/publish_javadoc11.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+if [[ -z "${CREDENTIALS}" ]]; then
+  CREDENTIALS=${KOKORO_KEYSTORE_DIR}/73713_docuploader_service_account
+fi
+
+if [[ -z "${STAGING_BUCKET_V2}" ]]; then
+  echo "Need to set STAGING_BUCKET_V2 environment variable"
+  exit 1
+fi
+
+#work from the git root directory
+pushd $(dirname "$0")/../../
+
+root_dir=$(pwd)
+
+python3 --version
+
+# install docuploader package
+python3 -m pip install --require-hashes -r .kokoro/requirements.txt
+
+# If DOCLET_VERSION is passed in (overriding version in shared-config)
+if [ -n "${DOCLET_VERSION}" ]; then
+  doclet_name="java-docfx-doclet-${DOCLET_VERSION}.jar"
+fi
+
+mvn -B -ntp \
+  -DtrimStackTrace=false \
+  -Dclirr.skip=true \
+  -Denforcer.skip=true \
+  -Dcheckstyle.skip=true \
+  -Dflatten.skip=true \
+  -Danimal.sniffer.skip=true \
+  -DskipTests=true \
+  -Djacoco.skip=true \
+  -T 1C \
+  install
+
+if [[ -z "${MODULE_LIST}" ]]; then
+  # Retrieve list of modules from aggregator pom
+  modules=($(mvn help:evaluate -Dexpression=project.modules | grep '<.*>.*</.*>' | sed -e 's/<.*>\(.*\)<\/.*>/\1/g'))
+else
+  modules=($(echo "${MODULE_LIST}" | tr ',' ' '))
+fi
+
+# TODO: Maps docs exclusion logic to be removed once we move to correct location on devsite. See b/262712184 and b/262600829
+excluded_modules=('gapic-generator-java-pom-parent' 'gapic-generator-java' 'gapic-generator-java-bom' )
+failed_modules=()
+
+declare -A cloud_rad_module_name
+cloud_rad_module_name+=( ["gax-java"]=gax ["api-common-java"]=api-common ["java-common-protos"]=proto-google-common-protos  ["java-iam"]=proto-google-iam-v1 )
```

Comment:
This may change the current [Cloud rad doc](https://cloud.google.com/java/docs/reference/proto-google-iam-v1/latest/overview) for `java-iam`. This is fine for now but we may need to double check the doc once this is released.